schema/release-schema.json

```diff
@@ -1369,7 +1369,7 @@
           "description": "(Deprecated in 1.1. Use transaction.payer instead.) The Organization Identifier for the organization from which the funds in this transaction originate. Expressed following the Organizational Identifier standard - consult the documentation and the codelist.",
           "$ref": "#/definitions/Identifier",
           "deprecated": {
-            "description": "This field has been replaced by the `transaction.payer` field to resolve ambiguity arising from 'provider' being interpreted as relating to the goods or services procured rather than the flow of funds between the parties.",
+            "description": "This field has been replaced by the `transaction.payer` field to resolve ambiguity arising from 'provider' being interpreted as relating to the goods, services and works procured rather than the flow of funds between the parties.",
```

Comment:
Was the conjunction changed from 'or' to 'and' deliberately?

Reply:
yes, again I thought it was possible for a mixture of these to be the focus of the transaction

Reply:
Per https://github.com/open-contracting/standard/pull/1530#discussion_r908510347, let's keep 'or'